### DISCUSSION TITLE
Trying to Run Local instance of Twake Chat

### DISCUSSION THREAD

**Discussion by gsriramsharath** [linagora/twake-on-matrix#2011] [Q&A]
the configuration data not found, this is what am getting when i cloned and ran the docker version of the given link [Git repo Twake chat](the https://github.com/linagora/twake-on-matrix/tree/v2.6.2) repo in my local machine which was built successfully but the it says config is missing how to address this issue any suggestions would be appreciated

**Comment by hoangdat** [comment]
Hi @gsriramsharath , please take a look on this document: https://github.com/linagora/twake-on-matrix/blob/main/docs/configurations/config_web_app_for_public_platform.md. if you only want to connect the app with your homeserver, you can ignore all config related to `public platform` or `saas`

**Reply by gsriramsharath** [reply]
thanks for the info so i have some doubts regarding the self hosted instance of twake chat so we need to build two repos which is  [https://github.com/linagora/twake-on-matrix/tree/v2.6.2](url) and  [https://github.com/linagora/ToM-server](url) used above two repos but can you share or give a link where i can get started with proper documentation or guide of the how to establish the Project that would be great when you say home server do you mean TOM-server ?
